### PR TITLE
Added support for multiple certs when creating a keystore

### PR DESCRIPTION
**Pull request by jkutner**
This brings the keystore creation in line with the truststore creation.

### CODE REVIEW

File: src/main/java/com/heroku/sdk/EnvKeyStore.java [hunk 1]
```diff
@@ -223,12 +223,18 @@ private static KeyStore createKeyStore(final Reader keyReader, final Reader cert
...
+
+    int i = 0;
+    X509Certificate certificate;
+
+    while ((certificate = parseCert(parser)) != null) {
+      ks.setKeyEntry(format("alias%d", i), key, password.toCharArray(), new X509Certificate[]{certificate});
+      i += 1;
+    }
+
+    parser.close();
```
Comment (holtonma) [inline src/main/java/com/heroku/sdk/EnvKeyStore.java]: Awesome, many thanks for this!  PR makes sense.

Let me see what I can do to pull this in with a SNAPSHOT jar, or vendor it, and verify it works in our Env with multiple certs.  Will let you know today, thanks again!

Comment (holtonma) [reply src/main/java/com/heroku/sdk/EnvKeyStore.java]: Built the `env-keystore` jar locally and set up our app to use the locally vendored jar.

Looks like the KeyStore is pulling in both Cert+IntermediateCert now @jkutner (alias0 and alias1 below).  We are doing a test on our Staging environment to see if this works all the way.  Will keep you posted. 
```
-------------------KEYSTORE-------------------------
keyStore alias: alias0
Type: X.509
PublicKey format: X.509
PublicKey algorithm: RSA
Cert:[
[
  Version: V3
  Subject: CN=sfdc-cb-rt-stg-ore.app.sfdc.sh, OU="0:sfdc-servicecloud-chatbot.sfdc-cb-rt-stg-ore;1:heroku.stg", O="salesforce.com, inc.", L=SAN FRANCISCO, ST=California, C=US
  Signature Algorithm: SHA256withRSA, OID = 1.2.840.113549.1.1.11

  Key:  Sun RSA public key, 2048 bits
 
  ...[wall of cert text redacted]...

]
keyStore alias: alias1
Type: X.509
PublicKey format: X.509
PublicKey algorithm: RSA
Cert:[
[
  Version: V3
  Subject: CN=salesforce.com Internal CA 1A, O="salesforce.com, inc.", C=US
  Signature Algorithm: SHA256withRSA, OID = 1.2.840.113549.1.1.11

  ...[wall of cert text redacted]...

]
----------------------------------------------------
```

File: src/main/java/com/heroku/sdk/EnvKeyStore.java [hunk 2]
```diff
@@ -223,12 +223,18 @@ private static KeyStore createKeyStore(final Reader keyReader, final Reader cert
...
-
     KeyStore ks = KeyStore.getInstance(DEFAULT_TYPE);
     ks.load(null);
-    ks.setKeyEntry("alias", key, password.toCharArray(), new X509Certificate[]{certificate});
+
+    int i = 0;
+    X509Certificate certificate;
+
+    while ((certificate = parseCert(parser)) != null) {
+      ks.setKeyEntry(format("alias%d", i), key, password.toCharArray(), new X509Certificate[]{certificate});
```
Comment (ml-sfdc) [inline src/main/java/com/heroku/sdk/EnvKeyStore.java]: @jkutner @holtonma It appears that we're taking the cert chain from the ENV var and attempting to decompose it into individual certs and insert each of them into the keystore as separate key entries (using the same private key(!)). We will never have keys to the Intermediate certs in the chain, as those are the signers. 

Our situation: Our servers trust "Company Internal Root CA" certificate. We have the key+ClientCert which is signed by Intermediate CA "Company Internal CA 1A". There's the Intermediate cert which says "Root CA trusts CA1", which we must send to allow the server to traverse down the chain of trust CA -> 1A -> ClientCert. Therefore, we need Java to send the 1A cert when it sends the ClientCert to the server for validation.

Comment (jkutner) [reply src/main/java/com/heroku/sdk/EnvKeyStore.java]: @ml-sfdc yea, i'm not sure what I was thinking with that change. I think #8 is a correct implementation. If you need me to cut a release for testing that I can.

**Pull request merged**